modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletedItems.testcase

```diff
@@ -366,22 +366,4 @@ definition {
 		WorkflowMetrics.viewDateRangeErrorMessage(errorMessage = "The end date cannot be earlier than the start date.");
 	}
 
-	@description = "LPS-86650 - Verify that the unexpected error state appear when unexpected error happen"
-	@ignore = "Test Stub"
-	@priority = 2
-	test UnexpectedErrorState {
-
-		// TODO LRQA-67648 UnexpectedErrorState pending implementation
-
-	}
-
-	@description = "LPS-86650 - Verify that, for numbers greater than 3 digits, use abbreviation standards considering localization specifications (example for English: replace 1,220 for 1.2K)"
```

Comment:
[LRQA-67645](https://liferay.atlassian.net/browse/LRQA-67645) This test is already covered in BE/FE tests